api/app/src/external/fhir/bundle/index.ts

```diff
@@ -0,0 +1,24 @@
+import { Bundle } from "@medplum/fhirtypes";
+import { capture } from "../../../shared/notifications";
+import { Util } from "../../../shared/util";
+import { makeFhirApi } from "../api/api-factory";
+
+export const MAX_FHIR_DOC_ID_LENGTH = 64;
+
+export async function postBundle(cxId: string, bundle: Bundle): Promise<void> {
+  const { log } = Util.out(`fhir.postBundle - ${cxId}`);
+  try {
+    const fhir = makeFhirApi(cxId);
+    const startTime = new Date().getTime();
+
+    await fhir.executeBatch(bundle);
+
+    const duration = new Date().getTime() - startTime;
+    log(`Successfully posted bundle to FHIR server, took ${duration}ms`);
```

Comment:
should we raise an issue & put a todo in the comments to capture all of these metrics somewhere?

Reply:
This file was not needed, removed.

Metrics are being sent to Cloudwatch from the lambdas, see `reportMetrics()`:

https://us-east-2.console.aws.amazon.com/cloudwatch/home?region=us-east-2#dashboards:name=Staging_Domain_metrics;start=2023-05-29T22:19:00Z;end=2023-05-29T22:52:00Z

We can defs send those from the app as well, but I'd review that with the use of Posthog... maybe what's user centric goes to Posthog and what's infra and not user specific we send to Cloudwatch?

Reply:
Ah yeah fair - wasn't sure if this was a one-off metric that wasn't being captured elsewhere, but this makes sense as-is